plugins/cleanupIDs.js

```diff
@@ -169,5 +168,5 @@ function generateID(currentID) {
  */
 function getIDstring(arr, params) {
     var str = params.prefix;
-    return str + arr.map(i => generateIDchars[i]).join('');
+    return str + arr.map(function(i) { return generateIDchars[i]}).join('');
```

Comment:
I love arrows and for of, but it's for this shitty uglifyjs, I'll try to figure a better way